WhaleSpotting/Models/ResponseModels/SightingResponseModel.cs

```diff
@@ -0,0 +1,53 @@
+using Newtonsoft.Json;
+using System;
+using WhaleSpotting.Models.DbModels;
+
+namespace WhaleSpotting.Models.ApiModels
+{
+    public class SightingResponseModel
+    {
+        [JsonProperty("sighted_at")]
+        public DateTime SightedAt { get; set; }
+        [JsonProperty("species")]
+        public string Species { get; set; }
```

Comment:
Attributes should have new lines before them, so can you do this throughout
```suggestion
        [JsonProperty("sighted_at")]
        public DateTime SightedAt { get; set; }
        
        [JsonProperty("species")]
        public string Species { get; set; }
```

Reply:
Done